WordPress/src/main/res/values/integers.xml

```diff
@@ -42,5 +42,5 @@
 
     <!--Site Creation-->
     <integer name="site_creation_snackbar_duration">5000</integer>
-
+    
```

Comment:
@gitstart ❓ Can we remove this blank space?

Reply:
@AjeshRPai 
Removed blank space.

Reply:
No problem @AjeshRPai. Reverted to exact `trunk` version. 
Please have a look now. Thank you!

Reply:
Thanks, @gitstart. All looks good 👍🏼 . I am triggering the CI for this Branch. Once it succeeds. I will merge it. Once again, thanks for working on this issue.